azure/availset.sh

```diff
@@ -8,6 +8,17 @@ fn azure_availset_create(name, group, location) {
 	azure availset create --name $name --resource-group $group --location $location
 }
 
+# azure_availset_exists checks if the given
+# availset exists inside the provided resource group.
+# returns "true" if it exists, "false" otherwise.
+fn azure_availset_exists(name, group) {
+        azure availset list $group | -grep -w $name
+        if $status == "0" {
```

Comment:
Sorry about the grep thing, did not know :-(

About: "but nash (and most shells) do not has boolean values", I just ask...why ? :-)